checker/tests/run-pass/tag_domain.rs

```diff
@@ -26,16 +26,17 @@ const SECRET_SANITIZER: TagPropagationSet = tag_propagation_set!(TagPropagation:
 
 type SecretSanitizer = SecretSanitizerKind<SECRET_SANITIZER>;
 
-pub fn test() {
-    let secret = 23333;
+pub fn test(secret: i32) {
+    precondition!(does_not_have_tag!(&secret, SecretTaint));
+    precondition!(does_not_have_tag!(&secret, SecretSanitizer));
 
     add_tag!(&secret, SecretTaint);
     verify!(has_tag!(&secret, SecretTaint));
     verify!(does_not_have_tag!(&secret, SecretSanitizer));
 
     let info = secret | 1;
     verify!(has_tag!(&info, SecretTaint));
-    verify!(does_not_have_tag!(&info, SecretSanitizer));
+    verify!(does_not_have_tag!(&info, SecretSanitizer)); //~ possible false verification condition
```

Comment:
It's not really the expected behavior. It's caused by the over-approximation of the `get_tags` function. In this case, we we would like `get_tags` to return a tag domain element that maps `SecretSanitizer` to `UnknownTagCheck(secret, SecretSanitizer)` instead of `Top`. I should have added a todo comment for this.

Reply:
Submitted another PR #552 for this issue.